crates/wash-lib/src/lib.rs

```diff
@@ -28,14 +28,21 @@ pub mod parser;
 #[cfg(feature = "start")]
 pub mod start;
 
+#[cfg(feature = "nats")]
 pub mod actor;
+#[cfg(feature = "nats")]
 pub mod capture;
 pub mod common;
+#[cfg(feature = "nats")]
 pub mod config;
+#[cfg(feature = "nats")]
 pub mod context;
+#[cfg(feature = "nats")]
 pub mod drain;
 pub mod id;
 pub mod keys;
 pub mod registry;
+#[cfg(feature = "nats")]
 pub mod spier;
+#[cfg(feature = "nats")]
```

Comment:
I think this can also work without the `nats` flag as the `Receiver` is just a tokio receiver 🤔

Reply:
it has a similar problem: a dependency to `actor` module which is behind `nats` flag.